packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -150,13 +150,13 @@ export class MeetWidget extends Component {
       this.props.checkWebRTCSupport(sparkInstance);
     }
     if (isWebRTCSupported) {
-      this.setup(this.props);
```

Comment:
What rule stops us from using `this`?

Reply:
The setup method doesn’t actually use this, so I moved it to a static method which means we can’t reference it with this.setup. Nothing stops us from using this.